src/pheval/cli_pheval_utils.py

```diff
@@ -244,3 +269,41 @@ def create_spiked_vcfs_command(
     if phenopacket_path is None and phenopacket_dir is None:
         raise InputError("Either a phenopacket or phenopacket directory must be specified")
     spike_vcfs(output_dir, phenopacket_path, phenopacket_dir, template_vcf_path, vcf_dir)
+
+
+@click.command()
+@click.option(
+    "--input",
+    "-i",
+    required=True,
+    metavar="FILE",
+    help="Path to the semsim file.",
+    type=Path,
+)
+@click.option(
+    "--output",
+    "-o",
+    required=True,
+    metavar="FILE",
+    help="Path to the semsim.h2.db converted file.",
+    type=Path,
+)
+@click.option(
+    "--subject-prefix",
+    "-s",
+    required=True,
+    metavar="FILE",
+    help="Subject Prefix that will be mapped to the database",
+    type=str,
+)
+@click.option(
+    "--object-prefix",
+    "-b",
+    required=True,
+    metavar="FILE",
+    help="Object Prefix that will be mapped to the database.",
+    type=str,
+)
+def semsim_convert(input: Path, output: Path, subject_prefix: str, object_prefix: str):
```

Comment:
Either:

Rename to account for the fact that we may need more convert methods in the future (`semsim_convert_exomiser`)

or

generalise method by adding a `-O/--output-format` parameter to indicate the format of the output (e.g. exomiser h2 db)

I prefer the second personally, but either one is fine.